### DISCUSSION TITLE
Disabled alias suffix for myself but enable for other users

### DISCUSSION THREAD

**Discussion by RequestPrivacy** [simple-login/app#1922] [Self-hosting question] [answered]
I try to run a self hosted Simple Login instance for my family and myself. 

As a treat to myself, I want to create aliases in the form of merchant@my-domain.com i.e. without alias suffix. So I set `DISABLE_ALIAS_SUFFIX=1` in `simplelogin.env` during the set-up.

But since family members also might sign up at "merchant" there's a potential conflict as they also might want to use merchant@my-domain.com.

How can I disable alias suffixes for me but for everyone else enable them? Can I set a flag in the db for each family member?

**Comment by springcomp** [comment, marked as answer]
@RequestPrivacy You cannot.

However, I created a similar setup for my family and preemptively reserved nice directory names for myself in advance.
That way, other family members will not be able to use those I have already registered.

**Reply by RequestPrivacy** [reply]
Thanks for answering. 

If I read correctly I don't have to worry whether two persons sign up with the same alias (`merchant@` in the above case) as simplelogin would prevent the second one from doing so? So the only inconvenience would be that the second person would need to choose a different name like `merchant1@` after it discovered that `merchant@` is already taken?

**Reply by springcomp** [reply]
@RequestPrivacy That is entirely correct.

**Reply by RequestPrivacy** [reply]
Thanks a lot for the confirmation, really appreciated!

**Comment by MuhammadNaeem42** [comment]
i will help you to setup this application 
contact me https://www.fiverr.com/s/9W3PNA
![simple](https://github.com/simple-login/app/assets/106573678/4a72f56a-10a8-4c0b-a92c-389568c98100)
